dependency/src/main/java/de/dagere/peass/execution/gradle/GradleBuildfileEditor.java

```diff
@@ -102,19 +102,32 @@ private void editGradlefileContents(final File tempFolder, final GradleBuildfile
    private void addDependencies(final GradleBuildfileVisitor visitor) {
       JUnitVersions versions = testTransformer.getJUnitVersions();
       boolean isExcludeLog4j = testTransformer.getConfig().getExecutionConfig().isExcludeLog4jSlf4jImpl();
+      boolean isAnbox = testTransformer.getConfig().getExecutionConfig().isUseAnbox();
+      final String peass = " // Added dynamically by Peass.";
```

Comment:
What does the peass-string to? As far as I see it, it is never used again.

Reply:
It's used in lines 115-120 as comments to show that the added changes to the build.gradle file were automatically added.